model/src/application/component/mod.rs

```diff
@@ -467,6 +470,8 @@ impl Serialize for Component {
                     + usize::from(text_input.required.is_some())
                     + usize::from(text_input.value.is_some())
             }
+            // We are dropping fields here but nothing we can do about that for the time being
```

Comment:
can we wrap this to 80 characters:
```suggestion
            // We are dropping fields here but nothing we can do about that for
            // the time being.
```